server/src/instant/db/model/triple.clj

```diff
@@ -164,7 +165,7 @@
             {:from 'eid+etypes-cte
              :join ['idents [:and
                              [:= 'idents/app-id app-id]
-                             [:= 'idents/etype 'idents/etype]]
+                             [:= 'idents/etype 'eid+etypes-cte/etype]]
                     'attrs [:= 'idents/id 'attrs/forward-ident]]
              :where [:= 'attrs/is_required true]
```

Comment:
Perhaps we can add an index to `is_required`.

Reply:
By this point we already narrowed down attributes to specific etype, so we are dealing with 10-ish attributes or so? I don’t think index here will help with anything

Reply:
Makes sense 🫡